### PR TITLE
Blog article: French translation "Instant Messaging: It is not a choice of the messenger"

### PR DESCRIPTION
**Pull request by Echolon**
Hello,

I started the French translation for #871

Please focus to the translation.
Veuillez vous concentrer sur la traduction.

### CODE REVIEW

**Comment by Echolon** [comment]
FYI @licaon-kter @linkmauve @Ppjet6

**Comment by Echolon** [comment]
Please review you own **comments**, I don't speak french and need to rely on your common sense :(

**Comment by licaon-kter** [comment]
This file has the name as the German one?

**Comment by Echolon** [comment]
> This file has the name as the German one?

Forgot to change, here we go:
https://github.com/xsf/xmpp.org/pull/875/commits/719ebf49be3134a29b021fb8c63d58d1ac8f4043

**Comment by Echolon** [comment]
@mathieui if you have motivation, this can also need a review

File: content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Contrairement à la situation actuelle où d'autres personnes vous suggèrent d'utiliser leur messager préféré, dans l'univers XMPP vous avez le libre choix de nombreux messagers - choisissez celui que vous aimez, pas celui que les autres aiment 
+jusqu'au prochain engouement pour l'internet. La différence est que, quelle que soit votre application de messagerie ou celle qu'utilisent vos amis, vous avez toujours le confort d'être sur le même réseau avec tous vos contacts, indépendamment de votre choix aujourd'hui. Je pense que c'est une vraie solution.
+
+### Il ne s'agit pas de l'application, mais de la technologie
+
+Réfléchissez à la technologie que vous comptez utiliser, puis décidez quel messager vous convient ou convient à votre environnement. XMPP et sa communauté ont accumulé une expérience variée, qui a été 
+utilisé dans de nombreux logiciels. Nous pensons que c'est une bonne solution pour la plupart des gens et des organisations, et pour l'Internet dans son ensemble - et il est ouvert à tous pour faire des innovations pour l'avenir.
```
Comment (mathieui) [inline content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication:42]: utilisé → utilisée (talking about the experience)

“et il est ouvert à tous pour faire des innovations pour l'avenir” → “et il reste ouvert à des innovations futures" ?

File: content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+author: Edward Maurer
+blog_id: blog
+category: misc
+---
+
+_This is the French translation of the original blog post._
+_C'est la traduction française de l'article original du blog._
+
+Plusieurs personnes m'ont récemment contacté pour me demander quel type de messagerie ils devraient utiliser maintenant - ils ont dit qu'ils ne comprenaient pas vraiment ce dont ils devraient s'inquiéter et s'ils devraient passer d'un des messagers connus à un autre. 
+Je me suis demandé comment répondre à cette question. Évidemment, j'aurais pu simplement plaider en faveur de XMPP (Extensible Messaging and Presence Protocol), mais j'ai ensuite pensé que cette réponse ne serait pas utile en soi. Souvent, les gens prennent une décision rapide concernant leur logiciel de communication et ce n'est généralement pas un choix fondé ; et ils finiront donc par passer à un autre messager plus tard.
```
Comment (mathieui) [inline content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication:13]: “de XMPP” → “d’XMPP”

**Comment by pulkomandy** [comment]
FYI translation was done on Linuxfr.org as well (where it's maybe easier to edit it and discuss the changes, if you have a linuxfr account): https://linuxfr.org/redaction/news/instant-messaging-it-s-not-about-the-app

Let's try to not duplicate efforts?

**Comment by Echolon** [comment]
> FYI translation was done on Linuxfr.org as well (where it's maybe easier to edit it and discuss the changes, if you have a linuxfr account): https://linuxfr.org/redaction/news/instant-messaging-it-s-not-about-the-app
> 
> Let's try to not duplicate efforts?

Hi @pulkomandy - that's great to hear. Unfortunately, I don't have an account there and I also don't speak French :( I personally would like have one official translation on the XSF page. Therefore I would need to ask for reviews here. If you still want to republish something else (direct translation, otherwise please point out to the reader), I'm fine with that.
Maybe it would be okay if one of you ready the correct I merge from @mathieui and tell me if you think that's good to go?

File: content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+date: 2021-01-24 06:30
+author: Edward Maurer
+blog_id: blog
+category: misc
+---
+
+_This is the French translation of the original blog post._
+_C'est la traduction française de l'article original du blog._
+
+Plusieurs personnes m'ont récemment contacté pour me demander quel type de messagerie ils devraient utiliser maintenant - ils ont dit qu'ils ne comprenaient pas vraiment ce dont ils devraient s'inquiéter et s'ils devraient passer d'un des messagers connus à un autre.
```
[the remark this answers is not in the document]
Comment (Echolon) [reply content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication:12]: Thanks, does this sentence makes sense?

> Contrairement à la situation actuelle où d'autres personnes vous suggèrent d'utiliser leur messagerie préféré, dans l'univers XMPP vous avez le libre choix de nombreux **messageries** - ...

**Comment by Echolon** [comment]
@mathieui thank you for your efforts!

**Comment by Echolon** [comment]
@pulkomandy If you consider further actions this PR please let me know so I can merge it into this here

**Comment by Echolon** [comment]
@pulkomandy do you want me to wait on feedback from your side?

**Comment by pulkomandy** [comment]
I don't know, I just wanted people to know that the work was being done twice. I gave some help on Linuxfr but I'm not in charge of anything there or here.

**Comment by Echolon** [comment]
Hi @Nyco,

are you working on that translation at jabberfr ? Maybe we can merge the content here and you post. Would be great to have  official translations in the blog for several languages.

@pulkomandy may you ask them there?

**Comment by Echolon** [comment]
Great thanks to: anubis, @mathieui, @Nyco, @pmaziere, @pulkomandy and yasabeau
I will merge asap and also tweet & toot

File: content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication [hunk 4]
```diff
@@ -67,21 +67,21 @@ d’Internet comme un tout, sans compter qu’elle reste ouverte à de futures i
...
 Il n’y a pas qu’une seule manière de concevoir des applications de communication. Faites le choix d’une technologie standardisée qui restera
-valable à long terme: pour cette raison, et pour beaucoup d’autres, nous recommandons l’utilisation d’XMPP!
+valable à long terme : pour cette raison, et pour beaucoup d’autres, nous recommandons l’utilisation d’XMPP!
 
-Si vous avez l’intention de choisir une solution pour la prochaine décennie, laissez-moi vous orienter vers plus de documentations:
+Si vous avez l’intention de choisir une solution pour la prochaine décennie, laissez-moi vous orienter vers plus de documentations :
 
-* Pour commencer, mettez le pied à l’étrier avec [le site de XSF](https://xmpp.org/getting-started/) [en anglais]
+* Pour commencer, mettez le pied à l’étrier avec [le site de XSF](https://xmpp.org/getting-started/) [en anglais] ;
 * Un article du blog de la XSF: [It's all about choices and control](https://xmpp.org/2015/01/its-all-about-choices-and-control/) (C’est une question de choix et de contrôle) [en anglais] par Laura, écrit et 2015 ;
```
Comment (pmaziere) [inline content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication:75]: XSF :

File: content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication [hunk 5]
```diff
@@ -67,21 +67,21 @@ d’Internet comme un tout, sans compter qu’elle reste ouverte à de futures i
...
-* [Modern XMPP](https://docs.modernxmpp.org/), un site web en anglais utile aux développeurs comme aux nouveaux arrivants pour se lancer dans XMPP
+* [Modern XMPP](https://docs.modernxmpp.org/), un site web en anglais utile aux développeurs comme aux nouveaux arrivants pour se lancer dans XMPP ;
 * Commentaires de la communauté XMPP:
```
Comment (pmaziere) [inline content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication:77]: XMPP :

File: content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication [hunk 6]
```diff
@@ -67,21 +67,21 @@ d’Internet comme un tout, sans compter qu’elle reste ouverte à de futures i
...
-    * [message sur Mastodon](https://pouet.april.org/@aprilorg/105520799332659637) de l’Association April
-    * [message sur Twitter](https://twitter.com/MovimNetwork/status/1351138046029279239) du réseau Movim
+    * [Eine Chance für XMPP](https://www.jabber.de/eine-chance-fuer-xmpp/) (Une chance pour XMPP) [en allemand], jabber.de ;
+    * [How to ensure your Instant Messaging solution offers users privacy and security.](https://www.erlang-solutions.com/blog/how-to-ensure-your-instant-messaging-solution-offers-users-privacy-and-security.html) (Comment s’assurer que votre solution de messagerie instantanée offre à vos utilisateurs une protection de leurs données personnelles et la sécurité) [en anglais], Erlang Solutions ;
+    * [message sur Mastodon](https://pouet.april.org/@aprilorg/105520799332659637) de l’Association April ;
+    * [message sur Twitter](https://twitter.com/MovimNetwork/status/1351138046029279239) du réseau Movim ;
 * [Ajout des Traducteurs français]:
```
Comment (pmaziere) [inline content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication:82]: français] :

**Comment by Echolon** [comment]
Thanks to everyone helping!

**Pull request merged**